code/modules/clothing/outfits/ert/frontiersmen_ert.dm

```diff
@@ -0,0 +1,59 @@
+/datum/outfit/centcom/ert/frontiersmen
+	name = "ERT - Frontiersman Basic"
+
+	head = /obj/item/clothing/head/beret/sec/frontier
+	mask = /obj/item/clothing/mask/gas/sechailer/minutemen
+	suit = /obj/item/clothing/suit/armor/vest/bulletproof/frontier
+	suit_store = /obj/item/gun/ballistic/rifle/boltaction
+	uniform = /obj/item/clothing/under/rank/security/officer/frontier
+	shoes = /obj/item/clothing/shoes/combat
+	gloves = /obj/item/clothing/gloves/combat
+	ears = /obj/item/radio/headset/pirate/alt
+	back = /obj/item/storage/backpack
+	belt = null
+	l_pocket = /obj/item/flashlight/seclite
+	r_pocket = /obj/item/tank/internals/emergency_oxygen/double
+
+	backpack_contents = list(/obj/item/ammo_box/a762=5, /obj/item/grenade/frag=1)
+
+	id_role = "Buccaneer"
```

Comment:
Buccaneer is a really pirate brained name, not frontiersman. Give them something more strikingly militaristic.